src/rgw/rgw_data_sync.cc

```diff
@@ -3149,6 +3173,7 @@ class InitBucketFullSyncStatusCR : public RGWCoroutine {
         if (check_compat) {
           // try to convert existing per-shard incremental status for backward compatibility
           yield call(new CheckAllBucketShardStatusIsIncremental(sc, sync_pair, num_shards, &all_incremental));
+          ldout(cct, 20) << "check for 'all incremental' in compatibility mode" << dendl;
```

Comment:
not sure this is helpful since the coroutine logging will already show `CheckAllBucketShardStatusIsIncremental`?

Reply:
ok. will remove